packages/quantic/cypress/integration/examples/example-quantic-facet.cypress.ts

```diff
@@ -0,0 +1,42 @@
+import {setupAliases} from '../../page-objects/example-quantic-facet';
+
+describe('example-quantic-facet', () => {
+  const quanticFacetUrl =
+    'https://sandbox-customization-power-8699-dev-17ae8f94da9.cs10.force.com/examples/s/quantic-facet';
```

Comment:
I did not handle the community URL dynamically yet.

Unfortunately `sfdx-cli` does not allow to simply retrieve a community URL. However the URL is returned when publishing the community.

So we could retrieve the community URL and then pass it to Cypress as the base URL. In addition we'll probably want to add a page for the `exampleSearch` component so all end-to-end tests are based on the community.